pkg/controllers/resourceapply/resourceapply.go

```diff
@@ -224,3 +294,45 @@ func applyDaemonSet(ctx context.Context, client coreclientv1.Client, recorder re
 	recorder.Event(required, corev1.EventTypeNormal, "Updated successfully", "Resource was successfully updated")
 	return true, nil
 }
+
+func applyPodDisruptionBudget(ctx context.Context, client coreclientv1.Client, recorder record.EventRecorder, requiredOriginal *policyv1.PodDisruptionBudget) (bool, error) {
+	required := requiredOriginal.DeepCopy()
+
+	existing := &policyv1.PodDisruptionBudget{}
+	err := client.Get(ctx, coreclientv1.ObjectKeyFromObject(required), existing)
+	if apierrors.IsNotFound(err) {
+		err = client.Create(ctx, required)
+		if err != nil {
```

Comment:
Nit, don't shadow the outer error, inline this please
```suggestion
		if err := client.Create(ctx, required); err != nil {
```

Reply:
This hasn't been done